apps/dashboard/src/contract-ui/tabs/nfts/components/mint-supply-tab.tsx

```diff
@@ -15,87 +15,96 @@ interface MintSupplyTabProps {
   tokenId: string;
 }
 
+// Intended for Edition contracts (not Edition Drop)
 const MintSupplyTab: React.FC<MintSupplyTabProps> = ({ contract, tokenId }) => {
   const trackEvent = useTrack();
+  const address = useActiveAccount()?.address;
   const {
     register,
     handleSubmit,
     formState: { errors },
     reset,
   } = useForm<{ to: string; amount: string }>({
-    defaultValues: { amount: "1" },
+    defaultValues: { amount: "1", to: address },
   });
 
-  const address = useActiveAccount()?.address;
-  const { mutate, isPending } = useSendAndConfirmTransaction();
-  const { onSuccess, onError } = useTxNotifications(
-    "Mint successful",
-    "Error minting additional supply",
-    contract,
-  );
+  const { mutateAsync, isPending } = useSendAndConfirmTransaction();
 
   return (
-    <div className="flex w-full flex-col gap-2">
-      <form
-        onSubmit={handleSubmit((data) => {
-          if (address) {
+    <form
+      className="flex w-full flex-col gap-2"
+      onSubmit={handleSubmit((data) => {
+        if (!data.to) {
+          return toast.error(
+            "Please enter the wallet address of the recipient.",
```

Comment:
Move this to form validation instead so user can't submit if inputs are not valid